src/bsp_types/requests/mod.rs

```diff
@@ -59,5 +61,9 @@ pub trait Request {
 pub trait CreateCommand {
     fn origin_id(&self) -> Option<String>;
 
-    fn create_command(&self) -> Command;
+    fn create_command(&self, root: PathBuf) -> Command;
+}
+
+pub trait CreateResult<T> {
+    fn create_result(&self, status_code: StatusCode) -> T;
 }
```

Comment:
Przerzuciłbym tego traita do modułu cargo_communication. Jak cchemy wydzielić crate'a, to będzie to niespójne logicznie. Podobnie z implementacją traitów.

Reply:
Jest to już przerzucone na nowym branchu

Reply:
A czemu nie na tym?

Reply:
Konkretniej to CreateResult zostanie wydzielony w tym PR. Patryk natomiast na nowym branchu zajmuje się odpowiednim tworzeniem komend (na razie nie są one poprawne), więc wszystkie zmiany ich dotyczące będą tam wprowadzone

Reply:
Poprawione